app/views/consent_responses/confirm.html.erb

```diff
@@ -6,3 +6,50 @@
 <% end %>
 
 <h1 class="nhsuk-heading-l">Check and confirm answers</h1>
+
+<div class="nhsuk-card" data-testid="consent">
+  <div class="nhsuk-card__content">
+    <h2 class="nhsuk-card__heading nhsuk-heading-m">
+      Consent
+    </h2>
+
+    <%= govuk_summary_list classes: 'app-summary-list--no-bottom-border' do |summary_list|
+      summary_list.with_row do |row|
+        row.with_key { "Consent" }
+        row.with_value { @draft_consent_response.consent.humanize }
+      end
+
+      summary_list.with_row do |row|
+        row.with_key { "For" }
+        row.with_value { @patient.full_name }
+      end
+
+      summary_list.with_row do |row|
+        row.with_key { 'By' }
+        row.with_value do %>
+          <%= @draft_consent_response.parent_relationship.humanize %>
+          <div class="nhsuk-u-margin-top-2 nhsuk-u-font-size-16">
+            <%= @draft_consent_response.parent_name %>
+            <% if @draft_consent_response.parent_phone.present? %>
+              <br />
+              Telephone: <%= @draft_consent_response.parent_phone %>
+            <% end %>
+            <% if @draft_consent_response.parent_email.present? %>
+              <br />
+              Email: <%= @draft_consent_response.parent_email %>
+            <% end %>
+          </div>
+        <% end
+      end
```

Comment:
Would it be worth using the consent card component? It looks like it should mostly work here.

Reply:
Just saw the commit message 😇  ... seems a pain to have the above section repeated, but guess we can 👞 🥫